### DISCUSSION TITLE
kernel最新分支linux-6.6.y，有build error

### DISCUSSION THREAD

**Discussion by wqiao-intel** [linuxdeepin/developer-center#7076] [Troubleshooting | 故障排查]
1.复现方式：
make allyesconfig
make -j$(nproc)

2.error log:
CC drivers/net/ethernet/pensando/ionic/ionic_lif.o
CC drivers/net/ethernet/pensando/ionic/ionic_rx_filter.o
CC drivers/net/ethernet/pensando/ionic/ionic_ethtool.o
In file included from drivers/net/ethernet/microchip/sparx5/sparx5_dcb.c:7:
./include/net/dcbnl.h:135:9: error: expected specifier-qualifier-list before ‘KABI_RESERVE’
135 | KABI_RESERVE(1)
| ^~~~~~~~~~~~
CC drivers/net/ethernet/pensando/ionic/ionic_txrx.o
CC drivers/net/ethernet/pensando/ionic/ionic_stats.o

3. 这是由最后一个提交引起的
9fc763cab25c (origin/linux-6.6.y) Init kabi reservation for driver and net system (#56)

4. 回退这个提交就可以编译过~

**Comment by wqiao-intel** [comment]
或者dcbnl.h 里面引用头文件 #include <linux/kabi.h>
也能解决

**Reply by BLumia** [reply]
deepin-community 组织下的项目请反馈到 deepin-community 组织下的对应项目仓库的 issue 面板中。linuxdeepin 组织只处理第一方项目的缺陷。

烦请移步： https://github.com/deepin-community/kernel/issues

另外欢迎 PR :)
